connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/internal/ConnectRuntimeConfig.scala

```diff
@@ -14,72 +14,43 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.spark.sql
+package org.apache.spark.sql.internal
 
 import org.apache.spark.connect.proto.{ConfigRequest, ConfigResponse, KeyValue}
 import org.apache.spark.internal.Logging
+import org.apache.spark.sql.RuntimeConfig
 import org.apache.spark.sql.connect.client.SparkConnectClient
 
 /**
  * Runtime configuration interface for Spark. To access this, use `SparkSession.conf`.
  *
  * @since 3.4.0
  */
-class RuntimeConfig private[sql] (client: SparkConnectClient) extends Logging {
+class ConnectRuntimeConfig private[sql] (client: SparkConnectClient)
```

Comment:
This looks like a breaking change at API level. 
- I'm wondering if this is okay, @hvanhovell ?
- In addition, please fix `@since 3.4.0` tag because `ConnectRuntimeConfig` only exists after 4.0.0.

Reply:
This class is not part of the public API. It is in internal, which is not public API. Regarding the tag I am not sure, that class has existed since 3.4.0. It just got renamed.

Reply:
To @hvanhovell . The previous `RuntimeConfig` is a documented public API.
- https://spark.apache.org/docs/3.5.2/api/scala/org/apache/spark/sql/RuntimeConfig.html